utils/src/main/java/com/indix/gocd/utils/store/S3ArtifactStore.java

```diff
@@ -34,7 +56,7 @@ public void put(String from, String to, ObjectMetadata metadata) {
     }
 
     public void put(PutObjectRequest putObjectRequest) {
-        putObjectRequest.setStorageClass(StorageClass.ReducedRedundancy);
+        putObjectRequest.setStorageClass(this.storageClass);
```

Comment:
Shouldn't we set this at the time the put object request gets built? or, we don't have the info there?